p2p/net/conn_mock.go

```diff
@@ -9,17 +9,10 @@ import (
 	"time"
```

Comment:
Mocks should be used only in tests so it is better to declare them only in a _test file.